### ISSUE TITLE
Bug: `numpy>=>=1.17`

### ISSUE THREAD

**Issue by alexlenail** [macs3-project/MACS#544]
```
pip install macs2

Collecting MACS2
  Using cached MACS2-2.2.7.1.tar.gz (2.1 MB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... error
  error: subprocess-exited-with-error

  × Preparing metadata (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [4 lines of output]
      Requirement already satisfied: numpy>=1.17 in /tmp/pip-build-env-25k996_o/overlay/lib/python3.9/site-packages (1.24.1)
      error in MACS2 setup command: 'install_requires' must be a string or list of strings containing valid project/version requirement specifiers; Expected end or semicolon (after name and no valid version specifier)
          numpy>=>=1.17
               ^
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.
```

When I download https://pypi.org/project/MACS2/#files I can verify that `/MACS2.egg-info/requires.txt` contains the string `numpy>=>=1.17`

**Comment by Umaarasu** [comment]
Hi, 
I am also having a similar issue. Could this be fixed?

× Preparing metadata (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [4 lines of output]
      Requirement already satisfied: numpy>=1.17 in /tmp/pip-build-env-880f00gu/overlay/lib/python3.8/site-packages (1.24.2)
      error in MACS2 setup command: 'install_requires' must be a string or list of strings containing valid project/version requirement specifiers; Expected end or semicolon (after name and no valid version specifier)
          numpy>=>=1.17
               ^
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

**Comment by Baboon61** [comment]
Same here, I am installing MACS2 inside a singularity environment. The version 2.2.7.1 was working fine some weeks ago, now I need to go back to 2.2.6 to make the installation works. This is not ideal for reproducibility purpose.

**Comment by dburkhardt** [comment]
This is a duplicate of https://github.com/macs3-project/MACS/issues/541

**Comment by philippadoherty** [comment]
Hello, thanks for your comments! This issue has been resolved in our [MACS2 v2.2.8 release](https://github.com/macs3-project/MACS/releases/tag/v2.2.8). Try it out!
